README.md

```diff
@@ -4,14 +4,11 @@ Windows Azure Active Directory IdentityModel Extensions for .Net
 IdentityModel Extensions for .NET provide assemblies that are interesting for web developers that wish to use federated identity providers for establishing the caller's identity.
 
 ## Versions
-Current version - 6.6.0 \
+
 Minimum recommended version - 5.5.0
```

Comment:
How about 6.5.1 as Minimum recommended version.

Reply:
It might be confusing for users as the latest AspNetCore handlers (3.1.5) depend on 5.5.0.
Perhaps we could remove "minimum recommended version" label as the recommended version is always the latest one :)